src/NUnitFramework/tests/Assertions/AssertThrowsAsyncTests.cs

```diff
@@ -212,6 +212,7 @@ public void BaseExceptionThrownAsync()
         }
 
         [Test]
+        [SetUICulture("en-US")]
         public void DerivedExceptionThrown()
```

Comment:
Failure reason: Test asserts a constant English exception message that doesn't match the localized exception message.